src/physics.F90

```diff
@@ -949,8 +949,8 @@ subroutine sample_target_velocity(nuc, v_target, E, uvw, v_neut, wgt, xs_eff, kT
           ARES_REJECT_LOOP: do
             ! perform Maxwellian rejection sampling
             xi = prn()
-            E_t = 16.0_8 * kT * xi**2
-            R = FOUR * xi * exp(ONE - E_t/kT)
+            E_t = 16.0_8 * xi * xi
```

Comment:
The variable name is a little misleading now since it's actually `E/kT`. Also, doesn't this mess up the calculation of `mu` and `v_target` down below?